flow-typed/forms-apps.js

```diff
@@ -104,22 +104,28 @@ declare type NewFormsListFormsApp = _NewFormsApp & {
   styles: FormsListStyles,
 }
 
-declare type NewTilesFormsApp = _NewFormsApp & {
-  type: 'TILES',
-  slug: string,
-  tiles: Array<FormTile | HrefTile | ContainerTile>,
-  styles: TilesStyles,
-}
-
-declare type Tile = {
+type Tile = {
```

Comment:
Should this be `_BaseFormsAppTile` as well?

Reply:
yep yep